### PR TITLE
IGNITE-22761 Disable WAL in RocksDB used to spill volatile Raft logs on disk

### PR DESCRIPTION
**Pull request by rpuch**
https://issues.apache.org/jira/browse/IGNITE-22761

### CODE REVIEW

File: modules/raft/src/main/java/org/apache/ignite/raft/jraft/storage/impl/RocksDbSpillout.java [hunk 1]
```diff
@@ -144,6 +144,7 @@ public RocksDbSpillout(
         this.groupEndBound = new Slice(groupEndPrefix);
 
         this.writeOptions = new WriteOptions();
+        this.writeOptions.setDisableWAL(true);
```
Comment (tkalkirill) [inline modules/raft/src/main/java/org/apache/ignite/raft/jraft/storage/impl/RocksDbSpillout.java:147]: How will we maintain consistency in saving commands?

Comment (rpuch) [reply modules/raft/src/main/java/org/apache/ignite/raft/jraft/storage/impl/RocksDbSpillout.java:147]: It is related to volatile storage. When starting a node, we recreate this RocksDB effectively destroying its state.

Comment (tkalkirill) [reply modules/raft/src/main/java/org/apache/ignite/raft/jraft/storage/impl/RocksDbSpillout.java:147]: It was simply not obvious to me from the name of the class that it was for volatile tables and it was said in the documentation at the very end.
Maybe I would rename the class to something like `RocksDbLogsNotSpillout`. 
Or at least indicate in the title of the problem that it is about volatile raft groups.

**Review by sashapolo — APPROVED** [review]

File: modules/raft/src/main/java/org/apache/ignite/raft/jraft/storage/impl/RocksDbSpillout.java (hunk 1 above)
Comment (rpuch) [reply modules/raft/src/main/java/org/apache/ignite/raft/jraft/storage/impl/RocksDbSpillout.java:147]: Included 'volatile' in the title of the Jira issue and PR

**Review by tkalkirill — APPROVED** [review]

**Pull request merged**